### ISSUE TITLE
Some questions about the code?

### ISSUE THREAD

**Issue by behome** [amirbar/speech2gesture#21]
I notice that the loss function of the discriminator is mse. In my view, cross-entropy is a commonly used loss function in binary classification tasks in discriminator. So I wanna know why? And I'm also confused in the output of the discriminator. The output of the discriminator is a vector with unequal length to the input, what dose it denote?
